modAionImpl/src/org/aion/zero/impl/cli/Arguments.java

```diff
@@ -131,6 +131,15 @@
             description = "execute kernel with selected port")
     private String port = null;
 
+    @Option(
+            names = {"--compact"},
+            arity = "1..2",
+            paramLabel = "<enabled> <slow_import> <frequency>",
+            description =
+                    "enable/disable compact during sync when one boolean parameter is given, or "
+                        + "enable when two values are provided for slow_import and frequency")
```

Comment:
should mention the time units in the description

Reply:
+1

Reply:
I didn't put the [ ] because square brackets indicates that the parameter is optional and its allowed to run ./aion.sh --force-compact  without passing any parameters, but in this case at least one parameter has to be provided.
The pipeline symbol indicates that its either 1 parameter or 2.

Reply:
i didn't put [ ] because square brackets indicate that the elements are optional and are allowed to occur zero times. But in this case at least one argument needs to be given. The pipe symbol indicates that its either 1 parameter or 2.